### PR TITLE
Create v1.2.0.md, draft of release announcements

### PR DESCRIPTION
**Pull request by elliothershberg**
We are planning on releasing in ~30 minutes, check it out and let us know any feedback or additional features worth highlighting 👍

### CODE REVIEW

File: website/release_announcement_drafts/v1.2.0.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+One of the core aspects of JBrowse 2 is that it is an extensible
+platform for biological visualization that can be extended with
+plugins. We are excited to introduce the first version of our
+plugin store, where we list the current external plugins that
+are available. Check it out [here](https://jbrowse.org/jb2/plugin_store).
+In the coming weeks, we will also be bringing this plugin store directly
+into the application, allowing plugin installation with the click of a button.
+Stay tuned!
+
+## Native SVG export
```
Comment (cmdcolin) [inline website/release_announcement_drafts/v1.2.0.md:14]: maybe remove term "native"?

File: website/release_announcement_drafts/v1.2.0.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+We're excited to introduce a new feature to JBrowse Web: built-in
+SVG export of track visualizations!
```
Comment (cmdcolin) [inline website/release_announcement_drafts/v1.2.0.md:17]: we have SVG export of linear genome views, with the more views to be added (could mention or not mention that more views to be added, but maybe mentioning that it is linear genome views might be useful)

File: website/release_announcement_drafts/v1.2.0.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+![](./img/svg_screenshot.png)
```
Comment (cmdcolin) [inline website/release_announcement_drafts/v1.2.0.md:19]: could consider this screenshot, i like that it shows bigwig and human data for example https://github.com/cmdcolin/jb2export/blob/master/img/1.png

Comment (elliothershberg) [reply website/release_announcement_drafts/v1.2.0.md:19]: Replaced the existing with this one, nice 😎

**Pull request merged**